index.rst

```diff
@@ -623,6 +623,13 @@ What's new in PyTorch tutorials?
    :link: intermediate/torch_compile_tutorial.html
    :tags: Model-Optimization
 
+.. customcarditem::
+   :header: (beta) Compiling the Optimizer with torch.compile
+   :card_description: Speed up the optimizer using torch.compile
+   :image: _static/img/thumbnails/cropped/generic-pytorch-logo.png
+   :link: recipes/compiling_optimizer.html
+   :tags: Model-Optimization
+
```

Comment:
Need to add this and the line 1056 to the [recipes_source/recipe_index.rst](https://github.com/pytorch/tutorials/blob/main/recipes_source/recipes_index.rst?plain=1) rather than here.